gnocchi/rest/api.py

```diff
@@ -1392,7 +1411,8 @@ def _search(self, **kwargs):
     @pecan.expose('json')
     def post(self, **kwargs):
         try:
-            return self._search(**kwargs)
+            json_attrs = get_json_attrs(kwargs)
```

Comment:
I'd probably put that outside of the `try/except` block for clarity. Otherwise one could think that this could raise an e.g. `IndexerException`, which should not be the case.